packages/api-explorer/src/components/Response/Result.jsx

```diff
@@ -9,52 +10,74 @@ const syntaxHighlighter = require('@mia-platform/syntax-highlighter');
-      }
-      {
-        !result.isBinary && isJson && (
-          <ReactJson
-            src={result.responseBody}
-            collapsed={1}
-            collapseStringsAfterLength={100}
-            enableClipboard={false}
-            theme="tomorrow"
-            name={null}
-            displayDataTypes={false}
-            displayObjectSize={false}
-            style={{
-              padding: '20px 10px',
-              backgroundColor: 'transparent',
-              color: colors.reactJson,
-              fontSize: '12px',
-              overflow: 'visible',
-              whiteSpace: 'pre-wrap',
-              wordBreak: 'break-all'
-            }}
-          />
-        )
-      }
-      {
-        !result.isBinary && !isJson && (
-          <pre className="tomorrow-night" style={notJsonStyle}>
-            <div className="cm-s-tomorrow-night codemirror-highlight">
-              {syntaxHighlighter(`${result.responseBody}`, result.type)}
-            </div>
-          </pre>
-        )
+      {result.isBinary && (
+        <div style={{display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
+          <div style={{padding: 8}}>
+            {FileSvg}
+          </div>
+          <span style={{color: '#fff'}}>
+            <FormattedMessage
+              id="api.response.binary"
+              defaultMessage="A {typeFile} was returned. Download file to view it"
+              values={{typeFile: result.type}}
+            />
+          </span>
+          <div style={{padding: 8}}>
+            <Button type={'primary'} onClick={() => window.open(result.responseBody)}>
+              <Icon type="download" />
+              <FormattedMessage id="code.download" defaultMessage="Download" />
+            </Button>
+          </div>
+        </div>
+      )}
+
+      {!result.isBinary && isJson && (
+        <ReactJson
+          src={result.responseBody}
+          collapsed={isCollapse === undefined ? 1 : isCollapse}
+          collapseStringsAfterLength={100}
+          enableClipboard={false}
+          theme="tomorrow"
+          name={null}
+          displayDataTypes={false}
+          displayObjectSize={false}
+          style={{
+            padding: '20px 10px',
+            backgroundColor: 'transparent',
+            color: colors.reactJson,
+            fontSize: '12px',
+            overflow: 'visible',
+            whiteSpace: 'pre-wrap',
+            wordBreak: 'break-all'
+          }}
+        />)
       }
+      {!result.isBinary && !isJson && (
+        <pre className="tomorrow-night" style={notJsonStyle}>
+          <div className="cm-s-tomorrow-night codemirror-highlight">
+            {syntaxHighlighter(`${result.responseBody}`, result.type)}
+          </div>
+        </pre>
+      )}
     </div>
   );
 }
 
 Result.propTypes = {
+  // eslint-disable-next-line react/require-default-props
+  isCollapse: PropTypes.bool,
```

Comment:
Set default props to false

Reply:
https://github.com/mia-platform/api-explorer/pull/35#discussion_r332044924

Reply:
If you set it to false in default, you can set the collapsed:
```
isCollapse || 1
```